README.md

```diff
@@ -1,6 +1,21 @@
-# Club Collabaration Backend
-
-Languages Used
-- Node.js
-- Express
-- MongoDB
+# Club Collabaration Backend
+
+Languages Used
+- Node.js
+- Express
+- MongoDB
+
+# Setup Instructions:
+
+## Installation
+
+```bash
+git clone https://github.com/iiitv/Club-Collabaration-backend/
+cd Club-Collabaration-backend
+yarn install
+```
+## Running
+
+```bash
+node app.js

```

Comment:
add a running script in package.json and give start command as `yarn start` or something

Reply:
Added running script in package.json, and updated it in README.md